### ISSUE TITLE
Owners can not break blocks

### ISSUE THREAD

**Issue by HeberonYT** [tastybento/ASkyBlock-Bugs-N-Features#97]
Hello,
Thanks for the recent update, the problem of the challenges in minecraft 1.7.10
Now I have another problem, recently I opened a skyfactory type public server, but some users can not break blocks, grass, wood, on their first island. Other users do not have this problem, so they are not permissions.
Sometimes it is solved by restarting the island, but this is very uncomfortable for new people.
I hope this report is not a duplicate.
Greetings and thanks for your work.

**Comment by tastybento** [comment]
New island is at 0,0? Maybe the issue is world spawn protection. Is there an error saying "Island protected" or do the blocks just not break? If the latter then it's definitely spawn protection. See the FAQ.

**Comment by HeberonYT** [comment]
It is not in 0.0
It can be any location, but it happens only on some occasions. And yes, that message appears in the chat "Island protected"
I have tried and I could not find the problem, but something interesting happened, when I do / island reset I create a new island but the previous one is not deleted from the map, it is still there normal, but I can not break anything and others either, then It just gets in the way.
When I reset it / island reset now only half the island is deleted, here I leave the image, and that same half continues with protection and I can not remove anything or kill the cow.

`http://i.imgur.com/EYuebdQ.png`

**Comment by tastybento** [comment]
Hmm, it sounds like the island distance changed. Can you pastebin.com your islands.yml and config.yml? I suspect the grid is not aligned.

**Comment by HeberonYT** [comment]
Of course, here it is, thanks for your attention.

**config.yml**  `https://pastebin.com/gK2xHEQE`
**islands.yml**  `https://pastebin.com/xUsFZr2C`

**Comment by tastybento** [comment]
Okay, can you give coords of the island where this problem happens. Also, when on that island, do

/asadmin info

and

/asadmin info (name of the owner)

an see if they are the same, or different.

**Comment by HeberonYT** [comment]
The island that was only half removed is in x:1400 z:399 y:120
The island that I eliminated and stayed complete is in x:-201 z:1 y:118

**/asadmin info**  = He says the island has no owner at this time. But I do not know why he does not erase it from the map.

**/asadmin info HeberonYT** = It shows several things in the chat, here below it leaves the image, the location of the island that says there is the new one that I have created at the moment, it is not the old island.
![screenshot_6](https://cloud.githubusercontent.com/assets/13698597/24811903/9f85e702-1b8d-11e7-90a4-5786b95fe806.png)
